File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        if len(nums)>=2:
            for i in range(len(nums)):
                if(nums[i]==0):
                    nums.append(nums[i])
                    nums.remove(nums[i])
                    

File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int right=nums.length-1;
        for(int i=0;i<right;i++){
            if(nums[i]==0){
                int temp=i;
                while(temp<right){
                    nums[temp]=nums[temp+1];
                    temp++;
                }
                nums[temp]=0;
                right--;
                i--;
            }
        }
    }
}